tests/e2e/models/generated_e2e_model_tests.cmake

```diff
@@ -0,0 +1,88 @@
+################################################################################
```

Comment:
Similar to #10928, do we want to check-in the generated .cmake file? The .cmake file can be called with
```
if(DEFINED IREE_BENCHMARK_SUITE_DIR)
  file(GENERATE ...)
  include(${CMAKE_CURRENT_BINARY_DIR}/generated_e2e_model_tests.cmake)
endif()
```

Reply:
It's actually not that simple because `file(GENERATE...` doesn't run at configuration step, so the `include(...` won't be able to find the file.

There are some more involved ways to do that (https://stackoverflow.com/questions/52556785/code-generator-generating-its-own-cmake-files-and-targets) But I haven't give them a try.

Tag #10928 to make sure we won't forget this one.